### PR TITLE
seed data added

### PR DESCRIPTION
**Pull request by sarahholder**

### CODE REVIEW

File: db/airports.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    "isInternational": true
+	},
+	"airport3": {
+		"uid": "Kenny",
+		"airportName": "John F. Kennedy Airport",
+		"imgUrl": "https://cdn11.bigcommerce.com/s-45hj43/images/stencil/500x659/products/4559/6793/CW_808_Statue_of_Liberty_with_New_York_Skyline__35635.1557789140.jpg?c=2&imbypass=on",
+		"city": "New York City",
+		"state": "NY",
+		"country": "US",
+		"airport code": "JFK",
```
Comment (zoeames) [inline db/airports.json:29]: this should be camel case = airportCode

File: db/airports.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+		"isInternational": false
+	},
+	"airport5": {
+		"uid": "Mary",
+		"airportName": "Edinburgh Airport",
+		"imgUrl": "https://cdn-image.departures.com/sites/default/files/1549985842/west-bow-street-architecture-edinburgh-scotland-EDINBURGH0219.jpg",
+		"city": "Edinburgh",
+		"state": "N/A",
+		"country": "SCT",
+		"airport code": "EDI",
```
Comment (zoeames) [inline db/airports.json:49]: airportCode

File: db/airports.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+		"isInternational": true
+	},
+	"airport8": {
+		"uid": "Kenny",
+		"airportName": "O’Hare International Airport",
+		"imgUrl": "https://images.discerningassets.com/image/upload/q_auto:best/c_limit,h_1000,w_1000/v1564957074/One_Night_in_Chicago_sm_vrx6cr.jpg",
+		"city": "Chicago",
+		"state": "IL",
+		"country": "US",
+		"airport code": "ORD",
```
Comment (zoeames) [inline db/airports.json:79]: airportCode

File: db/airports.json [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+		"isInternational": false
+	},
+	"airport9": {
+		"uid": "Zoe",
+		"airportName": "Leonardo da Vinci International Airport Rome",
+		"imgUrl": "https://www.fodors.com/wp-content/uploads/2018/10/HERO_UltimateRome_Hero_shutterstock789412159.jpg",
+		"city": "Rome",
+		"state": "N/A",
+		"country": "ITL",
+		"airport code": "FCO",
```
Comment (zoeames) [inline db/airports.json:89]: airportCode

File: db/airports.json [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+		"isInternational": true
+	},
+	"airport10": {
+		"uid": "Mary",
+		"airportName": "Cancun International Airport",
+		"imgUrl": "https://nationalpostcom.files.wordpress.com/2016/03/secrets-pools.jpg",
+		"city": "Cancun",
+		"state": "N/A",
+		"country": "MEX",
+		"airport code": "CUN",
```
Comment (zoeames) [inline db/airports.json:99]: airportCode

File: db/airports.json [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+		"isInternational": true
+	},
+	"airport11": {
+		"uid": "Zoe",
+		"airportName": "Kansai International Airport",
+		"imgUrl": "https://www.state.gov/wp-content/uploads/2019/04/Japan-2107x1406.jpg",
+		"city": "Izumisano",
+		"state": "",
+		"country": "JP",
+		"airport code": "KIX",
```
Comment (zoeames) [inline db/airports.json:109]: airportCode

**Comment by zoeames** [comment]
looks good 👍

File: db/airports.json (hunk 1 above)
Comment (sarahholder) [reply db/airports.json]: I so appreciate you pulling EVERY DETAIL I missed..

File: db/airports.json (hunk 2 above)
Comment (sarahholder) [reply db/airports.json]: You are going to kill me on PRs!

File: db/airports.json (hunk 3 above)
Comment (sarahholder) [reply db/airports.json]: I hope each comment I make dings you...

File: db/airports.json (hunk 4 above)
Comment (sarahholder) [reply db/airports.json]: Maybe I missed one of them... And you can reject my PR again

File: db/airports.json (hunk 5 above)
Comment (sarahholder) [reply db/airports.json]: Wow. Look. Same error.  I would have missed this one.

**Pull request merged**